### PR TITLE
Improve docs to help debugging

### PR DESCRIPTION
**Pull request by nvitucci**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-spark-28.surge.sh

Updated pages:
- [Writing to Neo4j](https://neo4j-docs-spark-28.surge.sh/spark/5/writing)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-spark-28.surge.sh

Updated pages:
- [Neo4j Connector for Apache Spark FAQ](https://neo4j-docs-spark-28.surge.sh/spark/5/faq)
- [Writing to Neo4j](https://neo4j-docs-spark-28.surge.sh/spark/5/writing)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-spark-28.surge.sh

Updated pages:
- [Neo4j Connector for Apache Spark FAQ](https://neo4j-docs-spark-28.surge.sh/spark/5/faq)
- [Writing to Neo4j](https://neo4j-docs-spark-28.surge.sh/spark/5/writing)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-spark-28.surge.sh

Updated pages:
- [Neo4j Connector for Apache Spark FAQ](https://neo4j-docs-spark-28.surge.sh/spark/5/faq)
- [Writing to Neo4j](https://neo4j-docs-spark-28.surge.sh/spark/5/writing)

**Comment by nvitucci** [comment]
@ali-ince This PR should include everything we have discussed so far without introducing any structure changes, which I will introduce in separate PRs for better clarity.

File: modules/ROOT/pages/writing.adoc [hunk 1]
```diff
@@ -646,12 +710,16 @@ Before the import starts, the following schema query is being created:
 CREATE INDEX ON :Person(surname)
 ----
 
-*Take into consideration that the first label is used for the index creation.*
+The name of the created index is `spark_INDEX_<LABEL>_<NODE_KEYS>`, where `<LABEL>` is the first label from the `labels` option and `<NODE_KEYS>` is a dash-separated sequence of one or more properties as specified in the `node.keys` options.
+In this example, the name of the created index is `spark_INDEX_Person_surname`.
+If the `node.keys` option were set to `"name,surname"` instead, the index name would become `spark_INDEX_Person_name-surname`.
+
+The index is recreated if it is already present.
```
Comment (ali-ince) [inline modules/ROOT/pages/writing.adoc:717]: I haven't checked what happens in this case, does it not use the `IF NOT EXISTS` clause in this case?

File: modules/ROOT/pages/writing.adoc [hunk 2]
```diff
@@ -670,8 +738,13 @@ Before the import starts, the code above creates the following schema query:
 CREATE CONSTRAINT FOR (p:Person) REQUIRE (p.surname) IS UNIQUE
 ----
 
-*Take into consideration that the first label is used for the index creation.*
+The name of the created constraint is `spark_NODE_CONSTRAINTS_<LABEL>_<NODE_KEYS>`, where `<LABEL>` is the first label from the `labels` option and `<NODE_KEYS>` is a dash-separated sequence of one or more properties as specified in the `node.keys` options.
+In this example, the name of the created constraint is `spark_NODE_CONSTRAINTS_Person_surname`.
+If the `node.keys` option were set to `"name,surname"` instead, the constraint name would become `spark_NODE_CONSTRAINTS_Person_name-surname`.
 
+The constraint is recreated if it is already present.
```
Comment (ali-ince) [inline modules/ROOT/pages/writing.adoc:745]: I'm not sure if this is correct. I think we explicitly put `IF NOT EXISTS` clause for constraints.

**Review by ali-ince — APPROVED** [review]
LGTM. Just left a couple of questions about index/constraint re-creation behaviour.

File: modules/ROOT/pages/writing.adoc (hunk 1 above)
Comment (ali-ince) [reply modules/ROOT/pages/writing.adoc:717]: Checked this, there is a logic to check if an equivalent index exists before creating the index, so shouldn't recreate one.

File: modules/ROOT/pages/writing.adoc (hunk 2 above)
Comment (ali-ince) [reply modules/ROOT/pages/writing.adoc:745]: Same here, so shouldn't recreate one.

File: modules/ROOT/pages/writing.adoc (hunk 1 above)
Comment (nvitucci) [reply modules/ROOT/pages/writing.adoc:717]: When I tried it, there was no error and the creation time of the index was updated. Have you tried as well?

**Pull request merged**